Source/Cake.DotNetTool.Module/DotNetToolPackageInstaller.cs

```diff
@@ -190,9 +198,9 @@ private List<DotNetToolPackage> GetInstalledTools(string toolLocation)
             var installedTools = isInstalledProcess.GetStandardOutput().ToList();
             var installedToolNames = new List<DotNetToolPackage>();
 
-            string pattern = @"(?<packageName>[^\s]+)\s+(?<packageVersion>[^\s]+)\s+(?<packageShortCode>[^`s])";

```

Comment:
This regex had a bug; \`s should be `\s` for whitespace match.

Reply:
Just noticed, it also didn't match more than one character. I guess it's not used.